scripts/ci/spack.yaml

```diff
@@ -7,6 +7,7 @@ spack:
   - ccache
   - cli11 +pic ~ipo
   - cmake
+  - covfie
```

Comment:
This is the answer to https://github.com/celeritas-project/celeritas/pull/2037/files#r2433716170